README.md

```diff
@@ -299,6 +300,82 @@ RestAssured.given(this.spec)
         .statusCode(200);
 ```
 
+#### WebTestClient based tests
+
+We've now added a convenience wrapper similar to `MockMvcRestDocumentationWrapper` for WebTestClient.
+The usage is similar to MockMVC, except that [com.epages.restdocs.apispec.WebTestClientRestDocumentationWrapper](restdocs-api-spec-webtestclient/src/main/kotlin/com/epages/restdocs/apispec/WebTestClientRestDocumentationWrapper.kt) is used instead of [com.epages.restdocs.apispec.MockMvcRestDocumentationWrapper](restdocs-api-spec/src/main/kotlin/com/epages/restdocs/apispec/MockMvcRestDocumentationWrapper.kt).
+
+To use the `WebTestClientRestDocumentationWrapper`, you will have to add a dependency to [restdocs-api-spec-webtestclient](restdocs-api-spec-webtestclient) to your build.
+```
+    import static org.springframework.restdocs.headers.HeaderDocumentation.headerWithName;
```

Comment:
Can we please have a less verbose example (like the ones above). You could still point to the real test in a link.